Public/Src/Cache/ContentStore/Library/Utils/IpcUtilities.cs

```diff
@@ -183,10 +200,13 @@ private static class EventWaitHandleAccessRules
         {
             public static EventWaitHandleAccessRule PublicSynchronizeAccessRule()
             {
-                var currentUser = UserUtilities.CurrentUserName();
+                // In CoreCLR, EventWaitHandle.TryOpenExisting() does not support setting access rights explicitly.
+                // Instead, a handle is always opened with EventWaitHandleRights.Synchronize | EventWaitHandleRights.Modify

```

Comment:
MIke: can you weigh in since you did the initial security work of the pipes and event handles.

Reply:
Reverted